### ISSUE TITLE
Skaffold clean up error

### ISSUE THREAD

**Issue by davidjsanders** [GoogleCloudPlatform/cloud-code-vscode#606]
djsanders@

I have a Cloud Code app running with Skaffold and Kustomize. 

**Expected Behaviour**: The app runs correctly in both Minikube and GKE Autopilot and cleans up on completion/stop.

**Actual Behaviour**: The app runs correctly in both Minikube and GKE Autopilot; however, on stopping the app in VS Code, an error is reported:

```
Cleaning up...
time="2022-06-23T19:32:56Z" level=warning msg="k8s/*.yaml did not match any file" subtask=-1 task=DevLoop
 - No resources found
```

I have to manually run `skaffold delete` from a command shell:
```
skaffold delete -p base
Cleaning up...
 - configmap "base-webapp-configuration" deleted
 - service "base-cr-exemplar-webapp" deleted
 - deployment.apps "base-cr-exemplar-webapp" deleted
```

The base Kustomize layer is...
```
resources:
  - configmap.yaml
  - deploy.yaml
  - service.yaml
namePrefix: base-
```
The skaffold.yaml is...
```
apiVersion: skaffold/v2beta26
kind: Config
metadata:
  name: webapp
build:
  artifacts:
  - image: cre-webapp
    docker:
      dockerfile: Dockerfile
profiles:
- name: base
  deploy:
    kustomize:
      paths:
      - manifests/base
```

The launch.json file is...
```
...
        {
            "name": "GKE: WebApp",
            "type": "cloudcode.kubernetes",
            "request": "launch",
            "skaffoldConfig": "${workspaceFolder}/webapp/skaffold.yaml",
            "profile": "base",
            "imageRegistry": "us-central1-docker.pkg.dev/xxx/docker-registry",
            "watch": true,
            "cleanUp": true,
            "portForward": true,
            "debug": [
                {
                    "image": "cre-webapp",
                    "port": 5679,
                    "containerName": "webapp",
                    "target": "app.py",
                    "sourceFileMap": {
                        "${workspaceFolder}/webapp": "/webapp"
                    }
                },
            ]
        },
...
```

Manifests are located in a tree...
```
| app
 -> manifests
   -> base
        - configmap.yaml
        - deploy.yaml
        - service.yaml
   -> dev
skaffold.yaml
... 
```

**Comment by davidjsanders** [comment]
This is related to [https://github.com/GoogleCloudPlatform/cloud-code-vscode/issues/519](https://github.com/GoogleCloudPlatform/cloud-code-vscode/issues/519)

My fix was to add...
```
deploy:
  kustomize:
    paths:
    - manifests/base
```
However, I'm not sure how this will work when running another profile (say dev or test)

**Comment by sushicw** [comment]
Hi David, thanks for reporting! We'll take a look and see if we can improve how this is handled.

**Comment by j-windsor** [comment]
The fix for this was released with Cloud Code v1.20.0.
